<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NormalComputationDlg</class>
 <widget class="QDialog" name="NormalComputationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compute normals</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="LocalSurfaceGroupBox">
     <property name="title">
      <string>Surface estimation</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>model</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="localModelComboBox">
        <property name="toolTip">
         <string>Local surface estimation model</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Plane</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quadric</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Triangulation</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>local radius</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="localRadiusFrame">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QDoubleSpinBox" name="radiusDoubleSpinBox">
           <property name="toolTip">
            <string>Local neighborhood radius (to estimate the surface)</string>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>0.000001000000000</double>
           </property>
           <property name="maximum">
            <double>999999.989999999990687</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="autoRadiusToolButton">
           <property name="text">
            <string>Auto</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="preferredOrientationGroupBox">
     <property name="title">
      <string>Preferred orientation</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="preferredOrientationComboBox">
        <property name="currentIndex">
         <number>4</number>
        </property>
        <property name="maxVisibleItems">
         <number>12</number>
        </property>
        <item>
         <property name="text">
          <string>+X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+Barycenter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>- Barycenter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+ (0,0,0)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>- (0,0,0)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Use previous normal</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NormalComputationDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NormalComputationDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
